src/GeneralStateTestsFiller/stEWASMTests/callSenderBalanceExceeds128BitsFiller.yml

```diff
@@ -0,0 +1,176 @@
+# Test for CALL with value
+callSenderBalanceExceeds128Bits:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 100000000000
+      code: ''
+      nonce: '0x4'
+      storage: {}
+    deadbeef00000000000000000000000000000001:
```

Comment:
I would suggest this test case sends value to an account and not a contract to simplify matters.